### ISSUE TITLE
Per-task progress bar broken in dashboard after 2.10

### ISSUE THREAD

**Issue by nsmith-** [dask/distributed#3490]
The following code sets up some tasks with two different names and runs them:
```python
import time
import numpy
from distributed import Client

def sleep1(t):
    time.sleep(t)

def sleep2(t):
    time.sleep(t)

if __name__ == '__main__':
    client = Client()
    a = client.map(sleep1, numpy.random.normal(loc=5, size=10))
    b = client.map(sleep2, numpy.random.normal(loc=5, size=10))
```
In distributed v2.9, the two functions each have a progress bar in the dashboard of appropriate length.  In v2.10, each have a progress bar that appears to be a duplicate, and is 20 items long rather than 10. It also advances in lock-step.
<img width="1073" alt="Screen Shot 2020-02-17 at 10 54 12 PM" src="https://user-images.githubusercontent.com/6587412/74705144-753ded80-51d8-11ea-8731-3dd3ced42592.png">

**Comment by jrbourbeau** [comment]
Thanks for raising an issue @nsmith-, I believe this should be resolved by #3322

**Comment by jacobtomlinson** [comment]
I'm going to close this out as #3322 has been merged and was included in the 2.11.0 release.

**Comment by nsmith-** [comment]
I confirm the issue is fixed in 2.11, thanks!
